Library/Homebrew/cmd/style.rb

```diff
@@ -111,12 +111,21 @@ def check_style_impl(files, output_type, options = {})
       args << "--only" << cops_to_include.join(",")
     end
 
+    has_non_formula = files.nil? || files.any? do |file|
+      puts "FILE: #{file}"
+      File.expand_path(file).start_with? HOMEBREW_LIBRARY_PATH
```

Comment:
Is the `expand_path` needed/warranted here?

Reply:
Yes. Consider the case where you're in `/usr/local/Homebrew/Library/Homebrew` and do `brew style ./descriptions.rb`. Since you're under the library, we want it to return true, but it won't unless the path is expanded to resolve that leading `./`.